docs/02-guides/server-side-events/create-an-integration.mdx

```diff
@@ -7,7 +7,135 @@ sidebar_position: 1
+
+## Write specific code for your integration
+
+All the event processing part will be done in the `handle` method. So we need to write all the code to send event to external services here.
+
+Since Webhook.cool is a basic services, we ca use a simple fetch to send the event :
+
+```ts title="app/server-evets/integrations/CoolWebhookIntegration.ts"
+import type { Integration } from "@front-commerce/core/server-events/integrations";
+
+export default class CoolWehbookIntegration implements Integration {
+  compatibleEvents: string[] | "*" = "*";
+
+  webhookId: string;
+
+  constructor(webhookId: string) {
+    this.webhookId = webhookId;
+  }
+
+  handle(messageId: string, payload: any) {
+    // highlight-start
+    fetch(`https://${this.webhookId}.webhook.cool`, {
+      method: "POST",
+      body: JSON.stringify(payload),
+    });
+    // highlight-end
+  }
+}
+```
+Remark that `webhookId` is used into the fetch called URL.
+
+:::info
+
+Here, we are posting the raw event details to the webhook, but feel free to manipulate it's data freely to adapt it to your specific integration.
+
+You'll find in the [FAQ](/docs/3.x/guides/server-side-events/faq#what-are-the-data-returned-by-the-server-events) the data format for the payload object
 
-- Create an integration
 :::
+
+## Register your integration
+
+Now that our integration is created, we have to register it in Front-Commerce.
+
+To do this, go to the `app/config/serverEvents.ts` file and register your new integration like this :
+
+```ts title="app/config/serverEvents.ts"
+import { ConsoleIntegration } from "@front-commerce/core/server-events/integrations";
+// highlight-next-line
+import CoolWehbookIntegration from "../server-events/integrations/CoolWehbookIntegration";
+
+export default {
+  redis: {
+    host:
+      process.env.FRONT_COMMERCE_SERVER_EVENT_REDIS_HOST ||
+      process.env.FRONT_COMMERCE_CLOUD_REDIS_HOST ||
+      process.env.FRONT_COMMERCE_REDIS_HOST ||
+      "127.0.0.1",
+    port:
+      process.env.FRONT_COMMERCE_SERVER_EVENT_REDIS_PORT ||
+      process.env.FRONT_COMMERCE_CLOUD_REDIS_PORT ||
+      process.env.FRONT_COMMERCE_REDIS_PORT ||
+      6379,
+  },
+  integrations: [
+    new ConsoleIntegration(),
+    // highlight-next-line
+    new CoolWehbookIntegration("XXXXXXXXXXXXXX"),
+  ],
+};
+```
+
+:::info
+
+You need to replace `XXXXXXXXXXXXXX` with your Webhook ID, you can find it on the [Webhook.cool](https://webhook.cool/) page
+
+![Webhook cool code](assets/webhook_cool_code.png)
+
+:::
+
+## Test your integration
+
+We are now ready to test your integration !
+
+Please be sure to read the [run local Server Event Worker guide](/docs/3.x/guides/server-side-events/faq#how-can-i-test-server-events-in-local-environment) before testing your integration.
+
+Then let's try to place an order and check out the [Webhook.cool](https://webhook.cool/) page. The OrderPlaced event is natively implemented in Front-Commerce.
```

Comment:
**thought:** shouldn't we use the Test event for this? It's easier than doing a full checkout (and soon we'll have simpler events triggered).

Reply:
I agree, we have the `/__front-commerce/test/events` endpoint for this. see: https://magento2.remixed.front-commerce.app/__front-commerce/test/events